src/main/java/org/opensearch/flowframework/transport/WorkflowRequest.java

```diff
@@ -62,13 +65,20 @@ public class WorkflowRequest extends ActionRequest {
      */
     private Map<String, String> params;
 
+    /**
+     * The timeout duration to wait for workflow completion.
+     * If null, the request will respond immediately with the workflowId.
+     */
+    @Nullable
```

Comment:
I see the timeout value defaults to -1, so this wont ever be null. We can remove the nullable annotation here and update the javadoc